components/automate-gateway/gateway/services.go

```diff
@@ -510,14 +504,11 @@ func (s *Server) ProfileTarHandler(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Length", contentLength)
 		w.Header().Set("Content-Type", "application/x-gzip")
 		w.Header().Set("Accept-Ranges", "bytes")
-		w.Write(data.GetData())
+		w.Write(data.GetData()) // nolint: errcheck
```

Comment:
It's not really your responsibility to fix this in this PR, but this seems like kind of a big deal...

Reply:
Now we know they are there :) Let's make a follow up ticket if we wanna error handle all of that!